app/src/main/java/ru/startandroid/organizer/home/HomeFragment.kt

```diff
@@ -28,6 +32,9 @@ class HomeFragment : android.app.Fragment() {
     lateinit var widgetEntityMapper: WidgetEntityMapper
     @Inject
     lateinit var widgetAdapter: WidgetAdapter
+    @Inject
+    @field:Named("weather")
```

Comment:
Why do we need Named annotation here? Now Dagger produces only one WeatherAPI object. Try to remove this annotation here and in NetworkModule. It should still work.